mathesar_ui/src/systems/table-view/header/header-cell/utils.ts

```diff
@@ -0,0 +1,33 @@
+import type { CellDataType } from '@mathesar/components/cell-fabric/data-types/typeDefinitions';
+import { SortDirection } from '@mathesar/stores/table-data';
+
+export function getSortingLabelForColumn(
```

Comment:
I'd like this method to be moved to `mathesar/components/sort-entry/utils` since it's not specific to tables.

`hasFkConstraint` needs to be optional.

Reply:
8b8e515ee

Reply:
@rajatvijay I would like to use the same labels in the SortEntry.svelte component. I believe it should be a small change and can be done in this PR itself.

Reply:
@rajatvijay seems to have handled in this in the last few commits.

I've added a commit https://github.com/centerofci/mathesar/pull/2246/commits/75cf8b74b66ffe86458f803c7252fa83314454a2 to resolve one last comment over the commits with respect to the label text.